src/cli/lib/runners/git.runner.ts

```diff
@@ -10,8 +10,11 @@ export class GitRunner extends GenericRunner {
 		super('git');
 	}
 
-	public async init({initialBranch = 'main', folderName}: InitOptions) {
-		const args = [`--initial-branch ${initialBranch}`, folderName];
-		await super.run({command: 'init', args});
+	public async init(folderName: string) {
```

Comment:
Maybe we could make use of the `createBranch` function in this one so we do the unit in one go

Reply:
I did it like that at first, but then it felt wrong to have both things coupled on a function called init, rather than having one for creating branches, what do you think?

Reply:
For me both approaches make sense. In fact, before we were creating the initial branch in the init method. So I don't mind if you decide one the other.

Reply:
Let's leave like that 👍